mobify/platform-scaffold
# [Integration Manager] Fix login commands

The Hybris connector work needed communicate a new value out of the `login` command. This converted the return value from a simple URL to a structure containing an `isCartMerged` key and a `href`. This PR updates all connectors to conform to that new return value.

 **JIRA**: N/A
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/893

## Changes
- As described above

## Todos

## How to test-drive this PR
- Switch to SFCC connector
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Login to a known SFCC account 
- Verify that you are redirected to the Account page after login and that you can access account information.
